rest/route/middleware.go

```diff
@@ -531,7 +553,7 @@ func (m *CommitQueueItemOwnerMiddleware) ServeHTTP(rw http.ResponseWriter, r *ht
 		if user.Id != *patch.Author {
 			gimlet.WriteResponse(rw, gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
 				StatusCode: http.StatusUnauthorized,
-				Message:    "Not authorized",
+				Message:    "Not authorized  commit queue author",
```

Comment:
Double space here

Reply:
oops i actually made these changes for testing so I'll get rid of it